redis/src/main/java/org/apache/calcite/adapter/redis/RedisServerMini.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.redis;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.embedded.RedisServer;
+
+/**
+ * RedisDataTypeTest.
+ */
+public class RedisServerMini {
+  private static JedisPool pool;
+  private static final int PORT = 6379;
+  private static final String HOST = "127.0.0.1";
```

Comment:
Does it belong to `src/main`?
It looks more like a `src/test` code

Reply:
> Does it belong to `src/main`?
> It looks more like a `src/test` code

Hi @vlsi, thank you very much, I want to ask you a question, how can I package ` RedisServerMini.java`  in the ` src / test`  directory into a jar package, and then use ` start.sh`  script to start this service. I'm not familiar with gradle. I've lost some information and can't find a better way.

Reply:
it could be:
a) `javaexec`-based task. Then the server could be started as `../gradlew runRedisMini`
b) JUnit-based class. Then the server could be started as `../gradlew test --tests package.Class`
c) Docker-based executable. Then the server could be started with TestContainers
d)...

Reply:
hi @vlsi Thank you very much, The PR is addressed.